doc/procedures/pre_commit.rst

```diff
@@ -0,0 +1,33 @@
+##########
+pre-commit
+##########
+
+``pre-commit`` is a tool to automatically run software tools, referred to as ``hooks``, when committing code to a versioning control repository like ``git``.
+See `the pre-commit website <https://pre-commit.com/>`_.
+
+To enable this:
+
+* Install pre-commit, for instance ``conda install -y pre-commit``.
+
+
+
+##################
+ts_pre_commit_conf
+##################
+
+``ts_pre_commit_conf`` is a tool to generate configuration files for the pre-commit hooks for the Telescope and Site SoftWare (TSSW) team at the Vera C. Rubin Observatory.
+
+These configuration files facilitate ``pre-commit`` to maintain ``black`` formatting, ``flake8`` compliance and ``isort`` imports sorting.
+If ``mypy`` is used, a configuration file can be generated for that as well.
+The configuration files are generated by using the ``generate_pre_commit_conf`` command.
+Since TSSW uses ``conda`` (or ``mamba``) for their software installations, a conda recipe was created for this project and the conda package was made available in the ``lsstts`` conda channel.
+To enable this:
+
+* Install ``ts_pre_commit_conf``, for instance ``conda install -y -c lsstts ts-pre-commit-conf``.
```

Comment:
I think the name of the package is actually `ts-pre-commit-config` (https://anaconda.org/lsstts/ts-pre-commit-config)...